doc/neps/nep-0027-zero-rank-arrarys.rst

```diff
@@ -200,10 +204,16 @@ remains on what should be the type of the result - zero rank ndarray or ``x.dtyp
 Since most if not all numpy function automatically convert zero-rank arrays to scalars on return, there is no reason for
 ``[...]`` and ``[()]`` operations to be different. 
 
+See SVN changeset 1864 (which became git changeset `9024ff0`_) for
+implementation of ``x[...]`` and ``x[()]`` returning numpy scalars.
+
+See SVN changeset 1866 (which became git changeset `743d922`_) for
```

Comment:
A nit: `commit` is more natural a word for git - I think `changeset` is a trac / SVN term

Reply:
fixed